src/main/java/ca/on/oicr/pineryreports/reports/impl/GazpachoProjectStatusReport.java

```diff
@@ -392,6 +400,32 @@ protected void collectData(PineryClient pinery) throws HttpResponseException, IO
     // reverse the "all" lists, as we want to display most recent items at the top
     Collections.reverse(dnaAllList);
     Collections.reverse(rnaAllList);
+
+    dnaStagnantList = getStagnant(dnaAllList);
+    rnaStagnantList = getStagnant(rnaAllList);
+  }
+
+  /**
+   * Get all the items created in the past six months which don't have recent activity (within the past month)
+   * 
+   * @param allReportItems list of all report items
+   * @return list of report items created in the past six months with no recent activity
+   */
+  private List<Entry<String, ReportItem>> getStagnant(List<Entry<String, ReportItem>> allReportItems) {
+    return allReportItems.stream().
+        filter(entry -> {
+          boolean stagnant = false;
+          try {
+            String lanesCompleted = entry.getValue().getNumLanes("Completed", null);
+            stagnant = "".equals(lanesCompleted)
+                && !dnaRecentList.contains(entry)
+                && !rnaRecentList.contains(entry)
+                && getDateTimeFormat().parse(entry.getValue().getStock().getCreatedDate()).toInstant().isAfter(SIX_MONTHS_AGO);
+          } catch (java.text.ParseException e) {
+            e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)